pythoncalculator/subtract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x, y):
+    return y - y

```

Comment:
I think this might be a tpyo and it should perhaps be....

```suggestion
    return x - y
```